src/app/[id]/page.tsx

```diff
@@ -38,47 +40,129 @@ export default function Page({params}: { params: { id: string } }) {
     <div>
       {catalogTitle ? (
         <div>
-          <h1 className="text-2xl font-bold mb-4">{catalogTitle.name}</h1>
+          <h1 className="text-4xl font-bold mb-4">{catalogTitle.name}</h1>
         </div>
       ) : (
         <div className="mb-4">Henter tittel ...</div>
       )}
 
-      <p className="mb-4">Serie-ID: {params.id}</p>
+      <p className="text-xl">Serie-ID: {params.id}</p>
+
+      <br></br>
 
       {localTitle ? (
-        <div>
-          {localTitle.vendor && <p>Avleverer: {localTitle.vendor}</p>}
-          {localTitle.contact_name && <p>Kontaktperson: {localTitle.contact_name}</p>}
-          {localTitle.contact_email && <p>E-post: {localTitle.contact_email}</p>}
-          {localTitle.contact_phone && <p>Telefon: {localTitle.contact_phone}</p>}
+        <div className="flex flex-col">
+          {localTitle.last_box ? (
+            <div className="flex flex-row items-center">
+              <p className="text-lg font-bold" >Eske til registrering: </p>
+              <p className="text-lg ml-2">{localTitle.last_box}</p>
+
+              <Button
+                endContent={<FaBoxOpen/>}
+                className="ml-3 font-bold"
+                size={'lg'}
+              >
+                Ny eske
+              </Button>
+
+            </div>
+          ) : (
+            <Button
+              endContent={<FaBoxOpen/>}
+              size={'lg'}
+              className="font-bold"
+              // TODO Add form for barcode (?) and/or link to box creation here (related to TT-1559)
+            >
+              Legg til eske
+            </Button>
+          )
+          }
```

Comment:
Bare en detalje, men de to knapper `Ny eske` og `Legg til eske` har egentlig samme funksjon.

Ville helst skrevet:

```
{localTitle.lastox && <info om last_box/>}
<Button>
  Legg til eske
</Button>
```

Reply:
Var mest for å få den inline, men er kanskje bedre at den er under. Ny commit ute, så et par småting etter diskusjon med Mona som jeg forbedret på også.